airbyte-integrations/connectors/destination-bigquery/build.gradle

```diff
@@ -4,9 +4,9 @@ plugins {
 }
 
 airbyteJavaConnector {
-    cdkVersionRequired = '0.1.4'
+    cdkVersionRequired = '0.1.5'
     features = ['db-destinations']
-    useLocalCdk = false
+    useLocalCdk = true
```

Comment:
Please CMIIW, my understanding of the steps are:

1. Wait for PR approval
2. Slash publish the CDK
3. Toggle this line (and the same for the other 2 dests) back to false
4. Merge the PR like usual

Reply:
looks right to me!